app/static/src/styles/comments.css

```diff
@@ -0,0 +1,15 @@
+.s26-comment-group {
```

Comment:
s26 is a prefix to show the sprint from the prototype - remove all s26 prefixes here

Reply:
yup my fault for not copying over from the prototype